lib/fog/key_manager/openstack.rb

```diff
@@ -41,6 +42,12 @@ class OpenStack < Fog::Service
       request :list_containers
       request :delete_container
 
+      #acl
+      request :get_acl
```

Comment:
please list the available requests (e.g. `get_secret_acl`) to make them discoverable, not the file names